YIF.Core.Service/Concrete/Services/SuperAdminService.cs

```diff
@@ -462,5 +462,16 @@ public async Task<ResponseApiModel<DescriptionResponseApiModel>> ChangeBannedSta
             }
             return result.Set(new DescriptionResponseApiModel(res), true);
         }
+
+        public async Task<ResponseApiModel<DescriptionResponseApiModel>> GetIoEAdminIdByIoEId(string ioEId) {
+            var result = new ResponseApiModel<DescriptionResponseApiModel>();
+            var admin = await _institutionOfEducationAdminRepository.GetByInstitutionOfEducationId(ioEId);
```

Comment:
Do you remember if IoE can have only one or few admins?

Reply:
The connection between IoE and IoEAdmin is One with Many, but as for me IoE should have only one Admin, what is more, if one with many is correct, than I'm not sure if the endpoint is appropriate for us, cause if it may return few admins, how Front will know which Id should be used?